data/profiles/demo.yaml

```diff
@@ -0,0 +1,413 @@
+hub: docker.io/istio
+tag: 1.1.4
+defaultNamespacePrefix: istio-system
+
+# Traffic management feature
+trafficManagement:
+  enabled: true
+  components:
+    namespace: istio-control
+
+    pilot:
+      common:
+        k8s:
+          env:
+          - name: GODEBUG
+            value: "gctrace=1"
+          replicaCount: 1
+          readinessProbe:
+            handler: {}
+# FIME: what's the problem here?
```

Comment:
1) s/FIME/TODO referencing user or github tracking issue.

2) Is this required or simplify copied from some existing file?

Reply:
Done, this is related to https://github.com/istio/istio/issues/15366, still investigating on the solution.